torch_geometric/nn/conv/cugraph/base.py

```diff
@@ -35,9 +36,10 @@ def reset_parameters(self):
     @staticmethod
     def to_csc(
         edge_index: Tensor,
-        size: Optional[Tuple[int, int]] = None,
+        size: Tuple[int, int] = None,
```

Comment:
This should remain optional if we allow `None` (which I think we should, so the rest of the logic looks good).